doc/user_guide/dynamo0p3.rst

```diff
@@ -1101,9 +1102,9 @@ Stencil metadata is written in the following format::
 
   STENCIL(type)
 
-where ``type`` may be one of ``X1D``, ``Y1D``, ``XORY1D``,
-``CROSS`` or ``CROSS2D``.  As the stencil ``extent`` (the maximum distance from the
-central cell that the stencil extends) is not provided in the metadata,
```

Comment:
Please could you add an example of a region stencil, as done for the X1D and Cross stencils below.

Reply:
Added an example of the REGION stencil to the user doc, see commit ae504ed6b45d1167d2d5c2c4b049a98e8f248954